.vscode/settings.json

```
@@ -0,0 +1,3 @@
+{
+	"typescript.tsdk": "node_modules/typescript/lib"
```

```suggestion
	"typescript.tsdk": "node_modules/typescript/lib",
	"editor.formatOnSave": true,
	"editor.defaultFormatter": "oxc.oxc-vscode",
```

maybe add oxc as workspace recommendations

---

What happens when someone doesn't have the VS Code extension installed?